### PR TITLE
docs(components): Update FormatTime docs

### PR DESCRIPTION
**Pull request by rodrigoeidelvein**
## Motivations

Updating FormatTime docs, it's web only.

### Added

- 

### Changed

- Updated FormatTime docs

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Review by dlrussel — CHANGES_REQUESTED** [review]
Looks good although in the code tab I'm seeing `invalid date` is this supposed to be this way? 
image

File: docs/components/FormatTime/Web.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          "@std-proposal/temporal": ["CivilTime"],
+        },
+      },
+    },
+  },
+} as ComponentMeta<typeof FormatTime>;
+
+const BasicTemplate: ComponentStory<typeof FormatTime> = args => (
+  <>
+    <FormatTime {...args} />
```
Comment (dlrussel) [inline docs/components/FormatTime/Web.stories.tsx:24]: I think the issues is with the way that we are passing in the args here 🤔

Comment (rodrigoeidelvein) [reply docs/components/FormatTime/Web.stories.tsx]: Exactly, it doesn't handle well dates and react component passed through the args object. Thanks for catching it!

**Review by dlrussel — APPROVED** [review]
looks good 💯

**Pull request merged**